src/scss/includes/vehicleSelector.scss

```diff
@@ -0,0 +1,77 @@
+.vehicleSelector {
+  display: flex;
+}
+
+.vehicleSelector-button {
+  cursor: pointer;
+  color: $grey-semi-darkness;
+
+  &--active {
+    color: $action-blue-base;
+  }
+
+  &::-moz-focus-inner {
+    border: 0;
+  }
+}
+
+@media (min-width: 641px){
+  .vehicleSelector {
+    padding: 4px;
+  }
+
+  .vehicleSelector-button {
+    width: 36px;
+    height: 36px;
+    border-radius: 50%;
+    margin-right: 12px;
+    font-size: 20px;
+
+    &:focus, &:hover {
+      color: $action-blue-base;
+      background-color: $action-blue-lighter;
+    }
+
+    &--active {
+      background-color: $action-blue-lighter;
+      border: 1px solid $action-blue-base;
+      box-shadow: 0 0 0 4px $action-blue-lighter;
+    }
+  }
+
+  .vehicleSelector-buttonLabel {
+    display: none;
+  }
+}
+
+@media (max-width: 640px){
+  .vehicleSelector {
+    justify-content: space-between;
+  }
+
+  .vehicleSelector-button {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-top: 8px;
+    min-width: 80px;
+
+    .vehicleSelector-buttonLabel {
+      margin: 2px 0 8px;
+    }
+
+    &--active {
+      .vehicleSelector-buttonLabel {
+        color: $primary_text;
+      }
+
+      &:after{
+        content:'';
+        width: 100%;
+        height: 4px;
+        background: $focus-gradient;
```

Comment:
according to mockups, it's a blue gradient. Can we define it in `colors.scss`, like `focus-gradient` ?

Reply:
Yeah…

Reply:
Done :)